tiledb/sm/c_api/tiledb.h

```diff
@@ -889,26 +889,46 @@ TILEDB_EXPORT int32_t tiledb_array_schema_has_attribute(
     int32_t* has_attr) TILEDB_NOEXCEPT;
 
 /**
- * Dumps the array schema in ASCII format in the selected output.
+ * Dumps the array schema in ASCII format in the selected file output.
  *
  * **Example:**
  *
  * The following prints the array schema dump in standard output.
  *
  * @code{.c}
- * tiledb_array_schema_dump(ctx, array_schema, stdout);
+ * tiledb_array_schema_dump_file(ctx, array_schema, stdout);
  * @endcode
  *
  * @param ctx The TileDB context.
  * @param array_schema The array schema.
- * @param out The output.
+ * @param out The output file.
  * @return `TILEDB_OK` for success and `TILEDB_ERR` for error.
  */
-TILEDB_EXPORT int32_t tiledb_array_schema_dump(
+TILEDB_EXPORT int32_t tiledb_array_schema_dump_file(
```

Comment:
You cannot rename an existing API; it would be a breaking change.

Reply:
You are right!